neo4j/src/main/java/org/hibernate/ogm/datastore/neo4j/Neo4jProperties.java

```diff
@@ -34,6 +34,31 @@
 	 */
 	public static final String SEQUENCE_QUERY_CACHE_MAX_SIZE = "hibernate.ogm.neo4j.sequence_query_cache_max_size";
 
+	/**
+	 * Socket inactivity timeout in milliseconds.
+	 */
+	public static String SOCKET_TIMEOUT = "hibernate.ogm.neo4j.client.socketTimeout";
+
+	/**
+	 * How long will we wait (in milliseconds) to get a connection?
+	 */
+	public static final String CONNECTION_CHECKOUT_TMEOUT = "hibernate.ogm.neo4j.client.connection_checkout_timeout";
+
+	/**
+	 * Time to live of the connection in the pool.
+	 */
+	public static final String CONNECTION_TTL = "hibernate.ogm.neo4j.client.connection_ttl";
+
+	/**
+	 * the timeout in millisecond to make an initial socket connection.
+	 */
+	public static final String ESTABLISH_CONNECTION_TIMEOUT = "hibernate.ogm.neo4j.client.establish_connection_timeout";
+
+	/**
+	 * Property for setting the password to connect with. Accepts {@code String}.
+	 */
+	public static String TTL_TIMEOUT = "hibernate.ogm.neo4j.client.socketTimeOut";
```

Comment:
There's something wrong here: the comment is wrong and the property is the same as SOCKET_TIMEOUT. Looks like a leftover as it's not used anywhere. I think we can remove it.